compiler/AST/type.cpp

```diff
@@ -216,6 +216,9 @@ const char* toString(Type* type, bool decorateAllClasses) {
       } else if (strncmp(at->symbol->name, drDomName, drDomNameLen) == 0) {
         retval = astr("domain", at->symbol->name + drDomNameLen);
 
+      } else if (startsWith(at->symbol->name, "_domain")) {
```

Comment:
Huh, how is it we've gone this long without a dtDomain symbol...?

What's the implication of using `startsWith()` rather than `strncmp()` here?

Reply:
It would match something like `_domain(int)` and replace it with `domain(int)`. Which I think would be good if it came up, but AFAIK it doesn't come up at present.

Reply:
(FWIW, I think it would be possible to write this fixup with checks on flags, but that did not feel worth it to me today)